Build error for Lecture-1

Bekomme den Fehler wenn ich Lecture-1 baue mit "make build_server ALGO_DESIGN=ufcd" :

Creating directories (server)
Building Lecture_eng.pdf (server)
Creating eng config
Building (1st iteration)
This is pdfTeX, Version 3.14159265-2.6-1.40.17 (TeX Live 2016/Debian) (preloaded format=pdflatex)
 restricted \write18 enabled.
entering extended mode
Makefile:127: recipe for target 'build_server_forced' failed
make: *** [build_server_forced] Error 1

--- a/Lecture-1/Makefile
+++ b/Lecture-1/Makefile
@@ -93,6 +93,7 @@ directories_server:
 # Create the index for all chapters
 build/$(ALGO_BUILD_CHAPTER_NAME).tex:
 	@echo "Reading chapters ($(ALGO_BUILD_CHAPTER_NAME))"
+	@mkdir -p build | true
 	@ls Chapter/$(ALGO_LANGUAGE)/*.tex | awk '{printf "\\input{%s}\n", $$1}' > build/$(ALGO_BUILD_CHAPTER_NAME).tex
 
 # ------------------------------------------------------------------------------

--- a/Lecture-1/Packages/LecturePackages_General.tex
+++ b/Lecture-1/Packages/LecturePackages_General.tex
@@ -1,7 +1,6 @@
 \usepackage[binary-units=true]{siunitx}
 
 \usepackage{adjustbox}
-\usepackage{tabitem-beamer}
 \usepackage{capt-of}
 
 \usepackage{tabularx, colortbl} % Tables

--- a/Lecture-10/Makefile
+++ b/Lecture-10/Makefile
@@ -93,6 +93,7 @@ directories_server:
 # Create the index for all chapters
 build/$(ALGO_BUILD_CHAPTER_NAME).tex:
 	@echo "Reading chapters ($(ALGO_BUILD_CHAPTER_NAME))"
+	@mkdir -p build | true
 	@ls Chapter/$(ALGO_LANGUAGE)/*.tex | awk '{printf "\\input{%s}\n", $$1}' > build/$(ALGO_BUILD_CHAPTER_NAME).tex
 
 # ------------------------------------------------------------------------------

--- a/Lecture-11/Makefile
+++ b/Lecture-11/Makefile
@@ -93,6 +93,7 @@ directories_server:
 # Create the index for all chapters
 build/$(ALGO_BUILD_CHAPTER_NAME).tex:
 	@echo "Reading chapters ($(ALGO_BUILD_CHAPTER_NAME))"
+	@mkdir -p build | true
 	@ls Chapter/$(ALGO_LANGUAGE)/*.tex | awk '{printf "\\input{%s}\n", $$1}' > build/$(ALGO_BUILD_CHAPTER_NAME).tex
 
 # ------------------------------------------------------------------------------

--- a/Lecture-12/Makefile
+++ b/Lecture-12/Makefile
@@ -93,6 +93,7 @@ directories_server:
 # Create the index for all chapters
 build/$(ALGO_BUILD_CHAPTER_NAME).tex:
 	@echo "Reading chapters ($(ALGO_BUILD_CHAPTER_NAME))"
+	@mkdir -p build | true
 	@ls Chapter/$(ALGO_LANGUAGE)/*.tex | awk '{printf "\\input{%s}\n", $$1}' > build/$(ALGO_BUILD_CHAPTER_NAME).tex
 
 # ------------------------------------------------------------------------------

--- a/Lecture-13/Makefile
+++ b/Lecture-13/Makefile
@@ -93,6 +93,7 @@ directories_server:
 # Create the index for all chapters
 build/$(ALGO_BUILD_CHAPTER_NAME).tex:
 	@echo "Reading chapters ($(ALGO_BUILD_CHAPTER_NAME))"
+	@mkdir -p build | true
 	@ls Chapter/$(ALGO_LANGUAGE)/*.tex | awk '{printf "\\input{%s}\n", $$1}' > build/$(ALGO_BUILD_CHAPTER_NAME).tex
 
 # ------------------------------------------------------------------------------

--- a/Lecture-14/Makefile
+++ b/Lecture-14/Makefile
@@ -93,6 +93,7 @@ directories_server:
 # Create the index for all chapters
 build/$(ALGO_BUILD_CHAPTER_NAME).tex:
 	@echo "Reading chapters ($(ALGO_BUILD_CHAPTER_NAME))"
+	@mkdir -p build | true
 	@ls Chapter/$(ALGO_LANGUAGE)/*.tex | awk '{printf "\\input{%s}\n", $$1}' > build/$(ALGO_BUILD_CHAPTER_NAME).tex
 
 # ------------------------------------------------------------------------------

--- a/Lecture-15/Makefile
+++ b/Lecture-15/Makefile
@@ -93,6 +93,7 @@ directories_server:
 # Create the index for all chapters
 build/$(ALGO_BUILD_CHAPTER_NAME).tex:
 	@echo "Reading chapters ($(ALGO_BUILD_CHAPTER_NAME))"
+	@mkdir -p build | true
 	@ls Chapter/$(ALGO_LANGUAGE)/*.tex | awk '{printf "\\input{%s}\n", $$1}' > build/$(ALGO_BUILD_CHAPTER_NAME).tex
 
 # ------------------------------------------------------------------------------

--- a/Lecture-2/Makefile
+++ b/Lecture-2/Makefile
@@ -93,6 +93,7 @@ directories_server:
 # Create the index for all chapters
 build/$(ALGO_BUILD_CHAPTER_NAME).tex:
 	@echo "Reading chapters ($(ALGO_BUILD_CHAPTER_NAME))"
+	@mkdir -p build | true
 	@ls Chapter/$(ALGO_LANGUAGE)/*.tex | awk '{printf "\\input{%s}\n", $$1}' > build/$(ALGO_BUILD_CHAPTER_NAME).tex
 
 # ------------------------------------------------------------------------------

--- a/Lecture-3/Makefile
+++ b/Lecture-3/Makefile
@@ -93,6 +93,7 @@ directories_server:
 # Create the index for all chapters
 build/$(ALGO_BUILD_CHAPTER_NAME).tex:
 	@echo "Reading chapters ($(ALGO_BUILD_CHAPTER_NAME))"
+	@mkdir -p build | true
 	@ls Chapter/$(ALGO_LANGUAGE)/*.tex | awk '{printf "\\input{%s}\n", $$1}' > build/$(ALGO_BUILD_CHAPTER_NAME).tex
 
 # ------------------------------------------------------------------------------

--- a/Lecture-4/Makefile
+++ b/Lecture-4/Makefile
@@ -93,6 +93,7 @@ directories_server:
 # Create the index for all chapters
 build/$(ALGO_BUILD_CHAPTER_NAME).tex:
 	@echo "Reading chapters ($(ALGO_BUILD_CHAPTER_NAME))"
+	@mkdir -p build | true
 	@ls Chapter/$(ALGO_LANGUAGE)/*.tex | awk '{printf "\\input{%s}\n", $$1}' > build/$(ALGO_BUILD_CHAPTER_NAME).tex
 
 # ------------------------------------------------------------------------------

--- a/Lecture-5/Makefile
+++ b/Lecture-5/Makefile
@@ -93,6 +93,7 @@ directories_server:
 # Create the index for all chapters
 build/$(ALGO_BUILD_CHAPTER_NAME).tex:
 	@echo "Reading chapters ($(ALGO_BUILD_CHAPTER_NAME))"
+	@mkdir -p build | true
 	@ls Chapter/$(ALGO_LANGUAGE)/*.tex | awk '{printf "\\input{%s}\n", $$1}' > build/$(ALGO_BUILD_CHAPTER_NAME).tex
 
 # ------------------------------------------------------------------------------

--- a/Lecture-6/Makefile
+++ b/Lecture-6/Makefile
@@ -93,6 +93,7 @@ directories_server:
 # Create the index for all chapters
 build/$(ALGO_BUILD_CHAPTER_NAME).tex:
 	@echo "Reading chapters ($(ALGO_BUILD_CHAPTER_NAME))"
+	@mkdir -p build | true
 	@ls Chapter/$(ALGO_LANGUAGE)/*.tex | awk '{printf "\\input{%s}\n", $$1}' > build/$(ALGO_BUILD_CHAPTER_NAME).tex
 
 # ------------------------------------------------------------------------------

--- a/Lecture-7/Makefile
+++ b/Lecture-7/Makefile
@@ -93,6 +93,7 @@ directories_server:
 # Create the index for all chapters
 build/$(ALGO_BUILD_CHAPTER_NAME).tex:
 	@echo "Reading chapters ($(ALGO_BUILD_CHAPTER_NAME))"
+	@mkdir -p build | true
 	@ls Chapter/$(ALGO_LANGUAGE)/*.tex | awk '{printf "\\input{%s}\n", $$1}' > build/$(ALGO_BUILD_CHAPTER_NAME).tex
 
 # ------------------------------------------------------------------------------

--- a/Lecture-8/Makefile
+++ b/Lecture-8/Makefile
@@ -93,6 +93,7 @@ directories_server:
 # Create the index for all chapters
 build/$(ALGO_BUILD_CHAPTER_NAME).tex:
 	@echo "Reading chapters ($(ALGO_BUILD_CHAPTER_NAME))"
+	@mkdir -p build | true
 	@ls Chapter/$(ALGO_LANGUAGE)/*.tex | awk '{printf "\\input{%s}\n", $$1}' > build/$(ALGO_BUILD_CHAPTER_NAME).tex
 
 # ------------------------------------------------------------------------------

--- a/Lecture-9/Makefile
+++ b/Lecture-9/Makefile
@@ -93,6 +93,7 @@ directories_server:
 # Create the index for all chapters
 build/$(ALGO_BUILD_CHAPTER_NAME).tex:
 	@echo "Reading chapters ($(ALGO_BUILD_CHAPTER_NAME))"
+	@mkdir -p build | true
 	@ls Chapter/$(ALGO_LANGUAGE)/*.tex | awk '{printf "\\input{%s}\n", $$1}' > build/$(ALGO_BUILD_CHAPTER_NAME).tex
 
 # ------------------------------------------------------------------------------

